### DISCUSSION TITLE
PCB Questions

### DISCUSSION THREAD

**Discussion by xprimnt** [RolandRust/HaniMandlWroom#5] [General]
(Die Übersetzung von ChatGPT ist unten)

Thank you for this excellent project. 

Apologies, this is the first time I will try ordering a PCB for a project like this. Any guidance you have would be helpful. 

Is there a place you recommend for ordering from? Is this something that you can order with the smaller components (such as SMD Mount resisters, diode, etc) already installed? I think I made my order wrong, but they paused it anyway due to two issues - risk of tab breaking and also that this was a multi board design and I would need to pay more. 

I suppose another question is that if someone had ordered multiple due to order quantity and had some left over, let me know! I’d like to build two but many places have minimums of five, so perhaps someone has a few laying around unused. 

Thanks in advance!

---

Vielen Dank für dieses ausgezeichnete Projekt.

Entschuldigung, dies ist das erste Mal, dass ich versuche, eine Leiterplatte (PCB) für ein solches Projekt zu bestellen. Jede Anleitung, die Sie haben, wäre hilfreich.

Gibt es einen Ort, den Sie für eine Bestellung empfehlen? Kann man so etwas mit den kleineren Komponenten (wie SMD-Widerstände, Dioden usw.) bereits installiert bestellen? Ich glaube, ich habe meine Bestellung falsch gemacht, aber sie wurde aufgrund von zwei Problemen pausiert – Risiko eines Tab-Bruchs und dass es sich um ein Multiboard-Design handelt und ich dafür mehr bezahlen müsste.

Ich nehme an, eine weitere Frage wäre, ob jemand aufgrund der Bestellmenge mehrere bestellt hat und noch welche übrig hat, lassen Sie es mich wissen! Ich möchte zwei bauen, aber viele Orte haben Mindestbestellmengen von fünf, daher hat vielleicht jemand ein paar ungenutzte übrig.

Vielen Dank im Voraus!

---

**Comment by xprimnt** [comment]
Fighting through ordering my first ever PCB with assembly. Any chance there is an associated Component and Pick and Place file I should be using? BoM is in the folder, but not a Centroid file. This is all new to me!

**Comment by RolandRust** [comment]
Hello, 
I ordered this project from JLCPCB. 10 multiboard circuit boards cost around $30 plus postage. The manufacturer's statement is correct that it is a multiboard design. Do this more often if a project consists of several circuit boards. Not ideal in terms of cost, but I have everything together on one circuit board and don't have to go looking for everything. I have never heard of the risk of a TAB breakage from a circuit board manufacturer. My bridges are actually the norm for use. I even left out the corresponding holes out of laziness. I also use NextPCB. I'm actually quite happy with the circuit boards for private use. 

Unfortunately, I have to disappoint you when it comes to assembling the circuit board. This circuit board is anything but optimized for machine bribery. This has to be done by hand. Otherwise it will be expensive. You could make the list of items and the pick and place file so that you could fill it mechanically. But all of these cheap providers have their own requirements for what it should look like. And if you don't use components that they have in stock, it will take you longer to do the mapping than it would take to assemble it manually.

I can't give an answer to the question of whether anyone still has circuit boards left. Don't know if anyone has already ordered it. Personally, I only have one left and I need it as a backup if something breaks.

**Comment by xprimnt** [comment]
Thank you for the detailed reply, it is much appreciated. I will amateurly follow in your footsteps and order the PCBs alone and work on my SMD component soldering skills. It’s a good project to fine tune them, anyway. I had wrongfully assumed that you had also ordered assembly and components, so I’m happy to adjust my plan - a good winter project when the eventual components arrive.
